### PR TITLE
chore: A1ODT-763 how to release a helm chart

### PR DESCRIPTION
**Pull request by SebastianBezold**

### CODE REVIEW

File: docs/guides/how-to-release-a-helm-chart.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+if you want to use the default configuration.
+
+1. Your helm charts need to be in the `/charts/<chart-name` directory of your repository
+2. GitHub pages need to be activated for your repository and point to a 'gh-pages' branch. Your GitHub organization
+   admins can activate that for you
+
+### Adding the chart release GitHub workflow
+
+As mentioned before, releasing the chart can be done via GitHub workflow. Following, there is an example workflow that
+uses the `chart-releaser-action` to publish package and publish your charts. You can use this example in your own
```
Comment (ricogu) [inline docs/guides/how-to-release-a-helm-chart.md:48]: could you please include a link for `chart-releaser-action`

**Review by ricogu — DISMISSED** [review]
just a small comment, i was aware of helm/chart-releaser-action@v1.4.0, but it is better to me to see a link to it

File: docs/guides/how-to-release-a-helm-chart.md (hunk 1 above)
Comment (SebastianBezold) [reply docs/guides/how-to-release-a-helm-chart.md:49]: Hi @ricogu. I added the link to the GitHub repo of chart releaser action on the first occurrence

**Review by ricogu — APPROVED** [review]

**Comment by ricogu** [comment]
thanks, approved

**Pull request merged**